setup.py

```diff
@@ -48,6 +48,8 @@
         'numpy',
         'scipy',
         'stsci.tools',
+	'pysiaf',
```

Comment:
Aligned now

Reply:
Still seeing a mix of spaces/tabs on these two lines.  This can break python in a difficult to debug way.

<img width="341" alt="image" src="https://user-images.githubusercontent.com/5471441/206734843-c6607dda-4d4d-43e7-a85d-ea1dff1acb65.png">

Reply:
I don't think it's an issue for continuation lines, however, I removed all tabs just to be sure